### ISSUE TITLE
Material dark theme variant in View not picked up in 14beta2

### ISSUE THREAD

**Issue by alvarezguille** [vaadin/flow#5781]
@eiswind commented on [Mon May 27 2019](https://github.com/vaadin/platform/issues/679)

```
@Route
@Theme(value = Material::class, variant = Material.DARK)
class LoginView(
```
is not rendered with the dark variant, whereas it works on other views with a layout

```
@Theme(value = Material::class, variant = Material.DARK)
class MainLayout : AppLayout(), RouterLayout, BeforeEnterObserver {
```

I noticed that its rendered dark as soon as a buttonclick happened on the page.

---

@pleku commented on [Mon May 27 2019](https://github.com/vaadin/platform/issues/679#issuecomment-496187929)

Thanks for the report!

This is a duplicate of https://github.com/vaadin/flow/issues/5715 and has already been fixed, to be released in the next 14 version.

---

@pleku commented on [Mon May 27 2019](https://github.com/vaadin/platform/issues/679#issuecomment-496189688)

Oops, actually I think this is not related to the ^ issue. Reopening and moving to vaadin/flow repository

**Comment by pleku** [comment]
Will check this after #5762 is fixed.

**Comment by manolo** [comment]
double checked after applying #5762 and it works.
